content/en/blog/2023-06-22-unikraft-gsoc-plat-rearch.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "re:Arch Unikraft"
+date = "2023-06-22T18:00:00+03:00"
+author = "Rares Miculescu"
+tags = ["GSoC'23", "plat re-arch"]
++++
+
+<img width="100px" src="https://summerofcode.withgoogle.com/assets/media/gsoc-2022-badge.svg" align="right" />
+
+## About Me
+
+I'm [Rareș Miculescu](https://github.com/rares-miculescu), a second year bachelor student at [University POLITEHNICA of Bucharest](https://upb.ro/en/).
+This is my first open source project that I am part of and I love it.
+Outside of Unikraft and university, I am obsessed with cars and Formula 1.
+
+## GSoC'23: re:Arch Unikraft
+
+### Motivation
+
+Unikraft provides specialization and strong modularity by strictly following the "everything is a library" concept.
+Despite this concept, Unikraft should have some architectural changes that can optimize and clearly define drivers and platform support.
+
+### Summary of Objectives
+
+The goal of this GSoC'23 project is to optimize and clearly define drivers and platform support.
+This can be achieved by solving the tasks from [plat-re-arch](https://github.com/orgs/unikraft/projects/36).
```

Comment:
```suggestion
This can be achieved by solving the tasks from [the `Platform Re-architecting` GitHub project](https://github.com/orgs/unikraft/projects/36).
```